src/utils/auth/resetPassword.ts

```diff
@@ -1,6 +1,7 @@
-import makeRequest from '/src/utils/services/makeRequest';
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
I think you could remove it for now. It's going to be dealt with when the integration of this function with the back-end is ready.